### PR TITLE
Fix missing links

### PR DESCRIPTION
**Pull request by yuki-js**

### CODE REVIEW

File: docs/conceptual/runtime/primitives.md [hunk 1]
```diff
@@ -71,4 +71,4 @@ There are an additional set of primitives that are assumed about a runtime built
 ### References
 
 - View the [`traits` defined in
-  `sr-primitives`](https://substrate.dev/rustdocs/master/sr_primitives/traits/index.html).
+  `node-primitives`](https://substrate.dev/rustdocs/master/node_primitives/traits/index.html).
```
Comment (JoshOrndorff) [inline docs/conceptual/runtime/primitives.md]: This link is still broken .Maybe you meant to use https://substrate.dev/rustdocs/master/sp_runtime/traits/index.html ?

Comment (riusricardo) [reply docs/conceptual/runtime/primitives.md]: Agree, I think that the correct one should be `sp-runtime`  https://substrate.dev/rustdocs/master/sp_runtime/traits/index.html

**Comment by yuki-js** [comment]
Oh, I made a wrong link.
Perhaps it may be https://substrate.dev/rustdocs/master/node_primitives/index.html from the content of this article.

**Comment by JoshOrndorff** [comment]
That's also a good one. I'd say both are relevant for this article. LEt's include both.

**Comment by yuki-js** [comment]
Fixed.

**Comment by JoshOrndorff** [comment]
Thanks :)

**Pull request merged**